inst/private/python_copy_vars_to.m

```diff
@@ -30,7 +30,6 @@
     L = { 'try:' ...
           sprintf('    %s = []', in) ...
           L{:} ...
-          '    octoutput_drv("PYTHON: successful variable import")' ...
```

Comment:
We need a different solution here b/c this will break popen2 ipc.  Can you just pass `false` for `te`?